src/site/content/en/blog/rakuten/index.md

```diff
@@ -0,0 +1,166 @@
+- Split all content into separate parts and lazy load below-the-fold HTML files.
+- Execute and load JavaScript on demand.
+- Identify slow JavaScript resources and optimize the loading process by using the async attribute on `<script>` tags and establishing early connections to important origins (Resource hints such as `dns-prefetch`, `preconnect`, and `preload`).
+- [Remove unused code](/remove-unused-code/), and [minify and compress code](/reduce-network-payloads-using-text-compression/).
+- Use a CDN.
+- [Control caching using Service Worker](/learn/pwa/caching/) with [Workbox](https://developer.chrome.com/docs/workbox/).
+
+#### Optimize images
+
+- [Lazy load below-the-fold images](/lazy-loading-images/).
+- [Optimize images with a CDN](/image-cdns/), deliver properly sized images, compress images, and adopt the right image formats for the job (WebP, SVG, Web Fonts).
+
+#### Optimize CLS
+
+- Use CSS [`aspect-ratio`](https://developer.mozilla.org/docs/Web/CSS/aspect-ratio) to reserve the required space for images while the images are loading.
+- Use CSS [`min-height`](https://developer.mozilla.org/docs/Web/CSS/min-height) to minimize layout shifts while elements are lazy loaded.
+
+### Performance measuring, analyzing, and monitoring
+
+#### Measuring
+
+Besides using [PageSpeed Insights](https://pagespeed.web.dev/) to audit their website, the team wanted to find a better way to know what the users are actually experiencing in the field. Therefore, Rakuten 24 decided to use the [web-vitals JavaScript library](https://github.com/GoogleChrome/web-vitals) to measure Core Web Vitals and other metrics in the field and send the data to inhouse analytics tool.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/o5Nd3IV6Yof3mfr3tVH3.jpg", alt="Rakuten 24's web vitals tracking integration flow. The first step is to integrate the web-vitals library by adding the script to the Rakuten 24 website. After that, web vitals can be measured from real user metrics, and the data is sent to Rakuten 24's in-house data collection tool.", width="800", height="495" %}
+</figure>
+
+#### Analyzing
+
+The team analyzed the collected field data to determine if there is any correlation between the Core Web Vitals and the key business metrics. They found out that the converted users tend to experience a better LCP than the users who didn’t convert.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/xWYjqq9sEJ2yQlDxl6fY.jpg", alt="A comparison of users who converted versus those who didn't by LCP. The user group who converted more frequently experienced a lower LCP.", width="800", height="391" %}
+</figure>
+
+The collected data also revealed that:
+
+- A good LCP can lead to an increase of up to **61.13% in conversion rate**, **26.09% in revenue per visitor**, and **11.26% in average order value**.
+- A good FID can lead to an increase of up to **55.88% in conversion rate** compared to the overall average data.
```

Comment:
@malchata Can `compared to the overall average data` be moved to a new line? Initially I separated this to a new line because this is the common part of both line 91 and 92 (A good LCP can ... compared to the overall average data, A good FID can .. compared to the overall average data). If you think it's better this way, that's fine too.